public/js/selfoss-db.js

```diff
@@ -1,6 +1,154 @@
 /**
  * db functions: client data repository
  */
+
+
+selfoss.dbOnline = {
```

Comment:
I do not understand the name. Under database, I usually imagine something that stores data.